### ISSUE TITLE
Cost Summary amends

### ISSUE THREAD

**Issue by anusenan** [unicef/etools-issues#95]
Expense total -> Expense Summary no number on the right of it
Expense Summary on top of the breakdown
Expenses summary pieces with their actual currency

**Comment by janpanchalingam1** [comment]
![image](https://cloud.githubusercontent.com/assets/26144740/24966276/7aed8e96-1f74-11e7-8c0b-69ebdef0856e.png)
TA can be submitted with multiple currencies which is an error (Waffle #94 ). Expense summary not showing currency of expenses.

**Comment by torbent** [comment]
Blocked by #217

**Comment by janpanchalingam1** [comment]
Passed.
![image](https://cloud.githubusercontent.com/assets/26144740/25537231/bc526d04-2c0b-11e7-8460-bb23b17b2ac6.png)

**Comment by kasfaw** [comment]
@janpanchalingam1 please provide complete QA description as discussed in our last meeting

**Comment by janpanchalingam1** [comment]
Cost Summary summing Traveler's DSA only. Correct calculation would sum Traveler's DSA and Expenses.

**Comment by janpanchalingam1** [comment]
Fail: cost summary summing only Traveler's DSA and does not sum expenses.
Browser: Chrome and IE
Workspace: Kyrgystan and Kenya
Link: https://etools-staging.unicef.org/t2f/edit-travel/173 
![image](https://cloud.githubusercontent.com/assets/26144740/25751985/db536ed2-3184-11e7-9422-70df1322923d.png)

**Comment by janpanchalingam1** [comment]
Fail: cost summary summing only Traveller's DSA and does not sum expenses.
Browser: Chrome and IE
Workspace: Kyrgystan
Link: https://etools-staging.unicef.org/t2f/edit-travel/182
![image](https://cloud.githubusercontent.com/assets/26144740/26001856/bc68051a-36fb-11e7-9baa-f86c3f1e636d.png)

**Comment by simon-the-sloth** [comment]
@janpanchalingam1 Actually now it's because of a misconfiguration in the database. All user related expense types has to have `user` as a vendor number and actually the used (`Air Ticket Cost`) has the following: `Traveler - Air Ticket Cost`. Someone has to go to the admin and make sure that all the user related expense types has the proper vendor number otherwise this error will pop up.

**Comment by torbent** [comment]
@anusenan @kasfaw we're putting on hold until we receive feedback here.

**Comment by kasfaw** [comment]
@torbent @simon-the-sloth there might be a major misunderstanding here:

traveler related expenses types (Air, Rail, Terminal, Other) should ALWAYS have a vendor number EQUAL TO to vendor number of the traveler. Keep in mind that the pupose of these expense types is to pay the traveler via his/her vendor number. If still not clear, let us know. 

We are hoping that on Invoice creation you have taken this into account...

**Comment by simon-the-sloth** [comment]
@kasfaw What you described here is what is happening, so all user related expenses get the user vendor number. But to determine this, the user related expense types has to have the vendor number `user` and later the code will replace this with the vendor number of the user. The problem here is that someone changed the vendor number of the user related expense types, and since the new one does not match, the backend thinks that those are travel agents.
My proposal would be to change those values back and then make the field read only on the admin if it's a user related expense type.

**Comment by torbent** [comment]
@kasfaw @anusenan have you made a decision on this?

**Comment by janpanchalingam1** [comment]
@kasfaw @anusenan after updating the admin as 'user' the cost summary continues to not calculate correctly:

**Comment by janpanchalingam1** [comment]
![image](https://cloud.githubusercontent.com/assets/26144740/26731265/7c6ed1a0-4781-11e7-9804-2a15b44c85fe.png)

**Comment by anusenan** [comment]
@janpanchalingam1  - could you share the expected calculation for the above screenshot? what is the formula to be applied?

**Comment by janpanchalingam1** [comment]
Traveller's DSA + Expenses (traveller expenses only NOT travel agency) = Total Cost Summary

**Comment by anusenan** [comment]
@simon-the-sloth  is this clear now?

**Comment by janpanchalingam1** [comment]
Pass
